### PR TITLE
Release version 4.9.0

### PR DESCRIPTION
**Pull request by bkabrda**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+# 4.9.0 / 2021-05-06
+
+* [IMPROVEMENT] Improvements for APT keys management.
+  * By default, get keys from keys.datadoghq.com, not the Ubuntu keyserver.
+  * Always add the `DATADOG_APT_KEY_CURRENT.public` key (contains key used to sign current repodata).
+  * Add `signed-by` option to all sources list lines.
+  * On Debian >= 9 and Ubuntu >= 16, only add keys to `/usr/share/keyrings/datadog-archive-keyring.gpg`.
+  * On older systems, also add the same keyring to `/etc/apt/trusted.gpg.d`.
+* [BUGFIX] Don't set `repo_gpgcheck=1` by default on RHEL/CentOS 8.1 and on custom repos. See [#352].
+* [BUGFIX] Change RPM key URLs to non-SNI versions to ensure the role continues to work with Pyton <= 2.7.9. See [#353].
```
Comment (KSerrania) [inline CHANGELOG.md]:
```suggestion
* [BUGFIX] Change RPM key URLs to non-SNI versions to ensure the role continues to work with Python <= 2.7.9. See [#353].
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
 CHANGELOG
 =========
 
+# 4.9.0 / 2021-05-06
+
+* [IMPROVEMENT] Improvements for APT keys management.
```
Comment (KSerrania) [inline CHANGELOG.md]:
```suggestion
* [IMPROVEMENT] Improvements for APT keys management. See [#351].
```

**Review by KSerrania — COMMENTED** [review]
Could you run `pimpmychangelog` (or another similar tool) to add links to issues & user profiles? They're missing since the 4.6.0 release.

**Comment by bkabrda** [comment]
@KSerrania done

**Review by KSerrania — APPROVED** [review]

**Pull request merged**